theme/palette/elements.less

```diff
@@ -7,3 +7,4 @@
 @sk-footer: @sk-mid-gray-light;
 @sk-background: @sk-gray-light;
 @sk-yellow: #fffce3;
+@sk-gold: #ffc600;
```

Comment:
Yes, #ffc600 is in the brand guidelines, generally used for Company Reviews (stars) :)